tests/framework/ab_test.py

```diff
@@ -97,6 +97,33 @@ def git_ab_test(
     return result_a, result_b, comparison
 
 
+def precompiled_binary_ab_test(
```

Comment:
I have some changes that simplify all test_vuln.py, but I'm not sure how do they play with this PR. I guess it can go through and then I can rebase it on top.